intellij-plugin-structure/tests/src/test/kotlin/com/jetbrains/plugin/structure/intellij/plugin/PluginVendorsTest.kt

```diff
@@ -9,6 +9,16 @@ import org.junit.Assert.assertTrue
 import org.junit.Test
 
 class PluginVendorsTest {
+  @Test
```

Comment:
see my comment about `com.intellij`

Reply:
Updated to check just the `vendor`, ignoring plugin ID completely.